qml/js/ProjectModel.js

```diff
@@ -340,7 +349,11 @@ function generateFileName(name, extension) {
 function regenerateCompilationResult()
 {
 	fileIo.deleteDir(compilationFolder)
```

Comment:
I think so. This function is called when mix is autogenerating the compilation result. this will never delete content added by the user.